packages/block-editor/src/components/index.js

```diff
@@ -104,7 +104,7 @@ export { default as BlockList } from './block-list';
 export { useBlockProps } from './block-list/use-block-props';
 export { LayoutStyle as __experimentalLayoutStyle } from './block-list/layout';
 export { default as BlockMover } from './block-mover';
-export { default as BlockPreview } from './block-preview';
+export { default as BlockPreview, useBlockPreview } from './block-preview';
```

Comment:
Hi, @andrewserong. Should we export this new hook as experimental for now? Similar to `useDisabled`?

Reply:
Yes, of course, that's a good idea. Thanks George, I've updated it in https://github.com/WordPress/gutenberg/pull/36431/commits/7c8af494d3d0310b82a345be678645a2b5e1e3ff to export as `__experimentalUseBlockPreview` which also removes it from the readme, too.

Reply:
Thanks, @andrewserong. I will try to do more detailed testing later today.